classes/announcement/Command.inc.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file clases/announcement/Command.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class announcement
+ *
+ * @brief Commands for modifying announcements
+ */
+
+namespace PKP\Announcement;
+
+use APP\core\Request;
+use PKP\core\Core;
+use PKP\plugins\HookRegistry;
+
+class Command
+{
+    public DAO $dao;
```

Comment:
(Do we want these `public`?)

Reply:
We still have a few cases where we go direct to the DAO for something (mostly the old upgrade scripts and the pubId stuff). But I agree it's something we should discourage.